2018-19/B08a Create care home name lookup.sps

```diff
@@ -29,7 +29,7 @@ GET DATA /TYPE=XLSX
    /READNAMES=on
    /ASSUMEDSTRWIDTH=32767.
 
-Rename Variables
```

Comment:
Check this, please. 

I think it can just be `Rename AccomPostCodeNo = CareHomePostcode.`

Either way, I think 'variables' is misspelt 😛